File: pinyougou-manager-web/src/main/webapp/js/service/sellerService.js
//服务层
app.service('sellerService',function($http){
    this.search = function (searchEntity,page,rows) {
        return $http.post("../seller/search.do?page="+page+"&rows="+rows,searchEntity);
    }
    this.findOne = function (sellerId) {
        return $http.get("../seller/findOne.do?sellerId="+sellerId);
    }
    this.updateStatus  =function (sellerId,status) {
        return $http.get('../seller/updateStatus.do?sellerId='+sellerId+"&status="+status);
    }
});

File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/SellerService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSeller;
import entity.PageResult;

public interface SellerService {
    Integer getSellerIdCount(String sellerId);

    void save(TbSeller tbSeller);

    PageResult search(TbSeller tbSeller, int page, int rows);

    TbSeller findOne(String sellerId);

    void updateStatus(String sellerId, String status);
}
